clients/drcachesim/tests/offline-kernel-opcode-mix.templatex

```diff
@@ -0,0 +1,7 @@
+Hello, world!
+Opcode mix tool results:
+.*: total executed instructions
+.*:.*
+.*
```

Comment:
Before this PR, syscall was at the top (all kernel instrs happened to be counted as the last encoding). So this test was useful for me currently. But to make it more useful for future changes, maybe we should check the presence of some kernel-only opcode?

Reply:
Used clac.